addons/website/static/tests/tours/media_iframe_video.js

```diff
@@ -48,6 +48,26 @@ registerWebsitePreviewTour("website_media_iframe_video", {
             trigger: ".modal-footer button:contains('Add')",
             run: "click",
         },
+        {
+            content: "Click on replace media",
+            trigger: "[data-replace-media='true']",
+            run: "click",
+        },
+        {
+            content: "Check that video url has protocol",
+            trigger: "#o_video_text",
+            run() {
+                if(!this.anchor.value.startsWith("https"))
```

Comment:
Sorry to bother you with this, but don't you have a linter to avoid these?
```suggestion
                if (!this.anchor.value.startsWith("https"))
```

Reply:
Yes, I do have lint, but it was somehow disabled. Done.